user/tests/wiring/no_fixture/time.cpp

```diff
@@ -270,11 +271,12 @@ test(TIME_16_TimeChangedEvent) {
 }
 
 test(TIME_17_RtcAlarmFiresCorrectly) {
-    if (Particle.syncTimePending()) {
-        waitFor(Particle.syncTimeDone, 60000);
-        assertTrue(Particle.syncTimeDone());
-        assertTrue(Time.isValid());
-    }
+    SCOPE_GUARD ({
+            Particle.connect();
```

Comment:
Nit: extra indentation.

We can perhaps even skip connecting here, `TIME_19_LocalTimeIsCloseToNtpTime` will connect us back anyway.